streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/Serdes.scala

```diff
@@ -30,12 +32,15 @@ object Serdes {
   implicit def JavaLong: Serde[java.lang.Long] = JSerdes.Long()
   implicit def ByteArray: Serde[Array[Byte]] = JSerdes.ByteArray()
   implicit def Bytes: Serde[org.apache.kafka.common.utils.Bytes] = JSerdes.Bytes()
+  implicit def byteBufferSerde: Serde[ByteBuffer] = JSerdes.ByteBuffer()
+  implicit def shortSerde: Serde[Short] = JSerdes.Short().asInstanceOf[Serde[Short]]
```

Comment:
```suggestion
  implicit def JavaShort: Serde[Short] = JSerdes.Short().asInstanceOf[Serde[Short]]
```

Reply:
Should this be `Short`? And we should additionally add
```
implicit def JavaShort: Serde[java.lang.Short] = JSerdes.Short()
```